lib/src/utils/scope.dart

```diff
@@ -0,0 +1,20 @@
+abstract class Scope {
+  const Scope(this.permission);
+
+  final String permission;
+
+  @override
+  String toString() => permission;
+}
+
+class EmailAddressScope extends Scope {
+  const EmailAddressScope() : super('r_emailaddress');
+}
+
+class LiteProfileScope extends Scope {
+  const LiteProfileScope() : super('r_liteprofile');
+}
+
+class MemberSocialScope extends Scope {
```

Comment:
To be honest i am not sure that we should put this scope since response for this scope is not mappable. Reason behind this is that LinkedIn rejected my application to have this scope as developer. It can mislead anyone who is using this package. At least we should put docs description ;]